todo_app/view_model.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime, timedelta
+
+
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def todo_items(self):
+        todo_items = []
+        for item in self._items:
+            if item.status == "To Do":
+                todo_items.append(item)
+            else:
+                continue
```

Comment:
These two lines don't do anything

Reply:
Ive removed all the else, continues. Thanks